netty/src/main/java/io/grpc/netty/NettyServerTransport.java

```diff
@@ -39,6 +39,8 @@
  */
 class NettyServerTransport implements ServerTransport {
   private static final Logger log = Logger.getLogger(NettyServerTransport.class.getName());
```

Comment:
Is the original `log` being used anymore after the change?

Reply:
I want to have both so it's more obvious to developers in the future that there's a difference between them.

Reply:
Then you might comment on the two fields what they are used for, and maybe add `@SuppressWarnings("unused")` to the first one.